hydra-cluster/src/CardanoClient.hs

```diff
@@ -32,17 +31,6 @@ buildScriptAddress script networkId =
   let hashed = hashScript script
    in makeShelleyAddress networkId (PaymentCredentialByScript hashed) NoStakeAddress
 
-queryStakePools :: NetworkId -> FilePath -> QueryPoint -> IO (Set PoolId)
-queryStakePools networkId socket queryPoint =
-  let query =
-        QueryInEra
-          BabbageEraInCardanoMode
-          ( QueryInShelleyBasedEra
-              ShelleyBasedEraBabbage
-              QueryStakePools
-          )
-   in runQuery networkId socket queryPoint query >>= throwOnEraMismatch
-
```

Comment:
Moved to `hydra-node`.

Cf discussion in https://github.com/input-output-hk/hydra-poc/discussions/453